### PR TITLE
Updates to the getting started, environment, and troubleshooting

### PR DESCRIPTION
**Pull request by jlperla**

### CODE REVIEW

File: lectures/getting_started_julia/getting_started.md [hunk 1]
```diff
@@ -41,138 +43,256 @@ While you will eventually use other editors, there are some advantages to starti
...
 
-Open a new terminal session and run
+This process will take several minutes to download and compile all of the files used by the lectures.
 
-```{code-block} none
-git clone https://github.com/quantecon/lecture-julia.notebooks
+```{attention}
+If the cursor is instead `(@v1.6) pkg>` then you may not have started the integrated terminal in the correct location, or you used an external REPL.  Assuming that you are in the correct location, if you type `activate .` in the package mode, the cursor should change to `(quantecon-notebooks-julia) pkg>` as it activates this project file.
+
+One benefit of using the integrated REPL is that it will set important options for launching Julia (e.g. the number of threads) and activate the local project files (i.e. the `Project.toml` file in the notebooks directory) automatically.  If you use an external REPL, you will need to set these manually.  Here you would want to run the REPL with `julia --project --threads auto`  to tell Julia to set the number of threads equal to your local machine's number of cores, and to activate the existing project.  See [here](starting-a-repl) for more details.
```
Comment (jlperla) [inline lectures/getting_started_julia/getting_started.md:234]: @Alim-faraji  is this where you saw the issue?

Comment (jlperla) [reply lectures/getting_started_julia/getting_started.md:234]: IT should be `repl_main` instead

Comment (Alim-faraji) [reply lectures/getting_started_julia/getting_started.md:234]: It is.

Comment (Alim-faraji) [reply lectures/getting_started_julia/getting_started.md:234]: Ok, I change it.

**Comment by jbrightuniverse** [comment]
Was this supposed to be left in here?
image

**Comment by jlperla** [comment]
@jbrightuniverse not supposed to be there, good catch. @Alim-faraji were you going to make a change here?

For now @jbrightuniverse why don't you remove that extra text and change it to whatever you think is clear.


And ghis is a good example of why we never put comments on changes to be made within the text itself. They should be in seaparte issues with links

**Comment by jbrightuniverse** [comment]
image
(in `getting_started`) I think the color identification doesn't match the image here?

**Comment by jlperla** [comment]
Sounds good, fix the color description!

**Pull request merged**